### ISSUE TITLE
Sprint 39 (2/1/24-2/14/24)

### ISSUE THREAD

**Issue by lmorris3** [department-of-veterans-affairs/va.gov-team#75018]
## PTO

Lauren: 2/2/24
Sophia: 2/6/24 (not PTO, but unavailable)
Sofia: 2/8-2/9

## Sprint 39 Goals

**Campaigns**
_High-level goal: Launch behavior change communications campaign._ 

Engineering:
- USiP PR, launch (blocked until SSiT OCTO gives word)

Design/Research:
- Deploy GovDelivery segmented emails (blocked, may unblock this sprint)

Data:
- Monitor sign in and call center data

Product:
- Distribute Call Center Guide
- Call Center and Help Desk Monthly Meeting presentation
- Send data reports to stakeholders
- Draft Project Planning doc for SSiT

**Stretch Goals**

**IPP**
_High level goal: Prepare for IPP Pilot Launch._

Engineering:
- Create a working case number
- Complete Login.gov IdP integration
- Develop sign in page for proofing agent app (consider breaking this out into fe and be)
- Develop sign in page for Veteran app (consider breaking this out into fe and be)
- Discovery: PII bundle
- begin VANotify discovery
- Complete LHDI setup steps
- Update VHIC features list

Research/Design:
- Wrap up Veteran UT synthesis
- Set up Proofing Agent Usability Tests
- Pilot planning: identify pilot site
- Draft pre-and post-pilot survey
- Draft PA training "curriculum"
- Design consistent footer for applications
- Design list of ID types that are not accepted
- Design improvements to Veteran intake 'home' screen

Product:
- Draft DIRA, review with Tom
- review Login.gov Charter with Tom, Login

**Stretch Goals**

- Complete LHDI onboarding (team)
- SME Interview synthesis
- Design A11y improvements to Veteran app

## What we want to demo at the end of the sprint

- Login.gov as IdP (Veteran Intake screen)
- Dashboard updates (tentative)
- IPP Pilot Planning: Site identified (dependent on VHRCs)
- Veteran Intake UT Insights
